plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-product-variations-controller.php

```diff
@@ -119,6 +119,8 @@ public function prepare_object_for_response( $object, $request ) {
 			'attributes'            => $this->get_attributes( $object ),
 			'menu_order'            => $object->get_menu_order(),
 			'meta_data'             => $object->get_meta_data(),
+			'name'                  => preg_replace( '/' . preg_quote( $object->get_title() . ' - ', '/' ) . '/', '', $object->get_name(), 1 ),
+			'parent_id'             => $object->get_parent_id(),
```

Comment:
Nice to both of these.

Reply:
> If developers wanted to use the parent_id property they would have to check which version of WooCommerce is installed, rather than rely that V3 is consistent.

Good point, and these haven't been the only changes. Originally given we are only adding properties and not changing the existing layout of the request we thought it be fine to just modify v3.
But given we have kind of taken advantage of that, a V4 makes more sense.
I will add this to our todo list.